<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0,0,0)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>741</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEditTask2">
       <property name="styleSheet">
        <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEditTask4">
       <property name="styleSheet">
        <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButtonTask3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(187, 38, 73)</string>
       </property>
       <property name="text">
        <string>Task3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineTask31">
         <property name="styleSheet">
          <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineTask32">
         <property name="styleSheet">
          <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)

</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineTask33">
         <property name="styleSheet">
          <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineTask34">
         <property name="styleSheet">
          <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineTask35">
         <property name="styleSheet">
          <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonTask1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(187, 38, 73)</string>
       </property>
       <property name="text">
        <string>Task1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonTask2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(187, 38, 73)</string>
       </property>
       <property name="text">
        <string>Task2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButtonTask4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(187, 38, 73)</string>
       </property>
       <property name="text">
        <string>Task4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditTask1">
       <property name="styleSheet">
        <string notr="true"> border-style: outset;
 border-width: 2px;
 border-color: rgb(47, 161, 190)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>653</width>
         <height>373</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(47, 161, 190)</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Hack'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Task1: AES&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Task2: SHA-512&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Task3: Dichotomy Method&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Task4: Dijkstra Algorithm&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>80</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(192, 97, 203)</string>
    </property>
    <property name="text">
     <string>Stat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
